test/unit3/test_counter.py

```diff
@@ -3,6 +3,39 @@
 import collections
 
 class CounterTests(unittest.TestCase):
+    def _mc_compare(self, result, expected):
```

Comment:
Thanks! I feel we explained often enough that the sorting algorithms in node change between versions. 😊

Reply:
Yes, the tests were clearly broken and should have been fixed like this a long time ago. :smile:

Reply:
tagging @bnmnetp who just asked a related question in #1066